lib/diaspora_federation/discovery/web_finger.rb

```diff
@@ -38,7 +38,7 @@ class WebFinger < Entity
       # @!attribute [r] alias_url
       #   @note could be nil
       #   @return [String] link to the users profile
-      property :alias_url, :string
+      property :alias_url, :string, default: nil
```

Comment:
Maybe we don't need to define a special `alias_url` property anymore and can rely on `additional_data[:aliases]` as the only method of defining aliases in the WebFinger (Xrd document)?